### DISCUSSION TITLE
Export Sensor Behaviour

### DISCUSSION THREAD

**Discussion by TikTok90** [BottlecapDave/HomeAssistant-OctopusEnergy#859] [Q&A] [answered]
I have a solar and battery setup together with a home mini.  For interest I'm putting together a chart to compare the output of the various meters. The import meter for the solar and the live data from the home mini are pretty close.  But I'm now trying to look at the export data, but I'm not sure what is happening with the Export Previous Accumulative Consumption entity.  I'm sure it just me not understanding but I'd either expect it to go up every day or reset to zero every day.  Instead I'm seeing this graph and don't understand what it is telling me.
![graph](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/138300803/300c495a-2cb1-4254-ae92-e4bac195fdc3)

**Comment by BottlecapDave** [comment, marked as answer]
The sensor represents the "latest" accumulative consumption reported by your meter to OE. This data is always at least 24 hours behind. These sensors grab their data straight from the API and do not include any data captured by the Mini. This is because this data is what is used to create your bills as per the law (the home mini is not a legal way of doing this), and function for anyone even if they don't have a Home Mini. Due to limitations with HA entities, I have not found a way to change historic data for a sensor, so the history view will always just be the final accumulative value.

Because of this, there are separate statistics sensors which could be used (this is what is recommended when setting up [energy dashboard](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/energy_dashboard/#for-electricity_1)). This splits the data down into hourly chunks (again the smallest block supported by HA).

**Reply by TikTok90** [reply]
Hi Dave - thanks for the explanation. I think what was confusing me was the way the steps up and down at around 1600 - but what you are saying is that it is just reporting once a day what yesterday's total was, which in my case is  about 1600 but could be anytime really.  Got it.
